saleor/graphql/account/mutations/staff.py

```diff
@@ -281,6 +281,23 @@ def _save_m2m(cls, info, instance, cleaned_data):
     def post_save_action(cls, info, instance, cleaned_input):
         info.context.plugins.staff_created(instance)
 
+    @classmethod
+    def get_instance(cls, info, **data):
```

Comment:
Hmm right, there is no need to send this notification, but I think it is nice to notify users that they were promoted to staff, that's why I feel this notification can be handy instead of creating a new one. What do you think?

Reply:
From one side it is nice to inform user that is promoted to staff, but from the second sending the set_password notification can be misleading as even if he didn't check an email, he will still be able to log in to the dashboard right?
But maybe using `set_password` here is better than adding separate notification. Would be good to hear what core team thinks about this.